apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSinkTest.java

```diff
@@ -225,7 +239,13 @@ public long getSequenceNumberForObject(String object) {
     @Nonnull
     @Override
     RdaChange<String> transformMessageImpl(String apiVersion, String s) {
-      return new RdaChange<>(1L, RdaChange.Type.UPDATE, s, Instant.now());
+      return new RdaChange<>(
+          1L,
+          RdaChange.Type.UPDATE,
+          s,
+          Instant.now(),
```

Comment:
It would be better to use the clock or a fixed time.

Reply:
This method gets mocked, so I've removed the contents